ansible/roles/prep.dirs/tasks/main.yml

```diff
@@ -50,6 +49,3 @@
     - "/etc/mock/fedora-*-x86_64.cfg"
 
 - include_tasks: repo.yml
-  with_items:
-    - "8"
-    - "9"
```

Comment:
I think we should leave it untouched. See below.

Reply:
Removing this allows us to only create configs for the os version being built for. If we leave this, we get centos8 configs generated when we are building centos9 rpms, I was trying to avoid this and only build for one specfic os version at a time.

Reply:
Alright, now it makes sense.